Assets/Scripts/OrchestraArmy/Room/Room.cs

```diff
@@ -3,13 +3,17 @@
 using System.Collections.Generic;
 using UnityEngine;
 using Random = UnityEngine.Random;
+using OrchestraArmy.Event;
+using OrchestraArmy.Event.Events.Enemy;
+
 
 namespace OrchestraArmy.Room
 {
-    public class Room
+    public class Room : IListener<EnemyDeathEvent>
```

Comment:
Er mist in deze klasse nog wel aardig wat commentaar

Reply:
nieuwe versie gecommit